content/posts/newsletter/2020-09-06.md

```diff
@@ -0,0 +1,134 @@
+
+The iOS and MacOS client Monal got a [new push server](https://monal.im/blog/monal-push-server-upgrade/) and received improvements for Message Archive Management. This is one of the many changes to arrive in future releases. Be prepared and get a glimpse in the [Monal Beta Testflight](https://monal.im/blog/news-betas-up-for-4-8/).
+
+The Python SDK for GE WiFi-enabled kitchen appliances, [gekitchen](https://github.com/ajmarks/gekitchen), made its first GitHub release. The primary goal is to use it to power integrations for [Home Assistant](https://www.home-assistant.io/), though that probably needs to wait for some new entity types. It is using XMPP as a mean to authenticate, since it is what the The GE SmartHQ app uses to communicate with devices.
+
+Hello [BeagleIM 4.0 and SiskinIM 6.0](https://tigase.net/beagleim-4.0-and-siskin-6.0-released/)! Their iOS and MacOS clients now speak Mediated Information eXchange (MIX) [(XEP-0369, XMPP’s new group chat protocol)](https://xmpp.org/extensions/xep-0369.html). VoIP connectivity has been improved with Jingle Message Initiation (XEP-0353), Last Message Correction ([XEP-0308](https://xmpp.org/extensions/xep-0308.html)) is available for editing messages which were already sent, and other features like Message Retraction ([XEP-0424](https://xmpp.org/extensions/xep-0424.html)) and quick replies for quoting were introduced. Support for CAPTCHA forms has been added as well: Extended In-Band Registration ([XEP-0158](https://xmpp.org/extensions/xep-0158.html)) is invoked for additional verification during account registration. Of course there are many more changes and bugfixes! XMPP on iOS and Mac is getting traction after all!
+
+## Servers
+
+No chance for SPAM: Openfire presents their new ['Spam blacklist' plugin](https://discourse.igniterealtime.org/t/new-openfire-plugin-to-help-reduce-spam/88730) which enables reporting for users. Everyone, please report potential spammers! Openfire’s Monitoring plugin has been released in version 2.1.0, featuring full text search for personal archive messages. Last but not least there is a new [beta online for Smack 4.4.0 (Smack 4.4.0-beta2)](https://discourse.igniterealtime.org/t/smack-4-4-0-beta2-released/88804).
+
+The Prosody XMPP server team have been busy this month with several exciting things. Firstly, the [Prosody 0.11.6 release announcement](https://blog.prosody.im/prosody-0.11.6-released/). They also published a suite of new modules to allow [simple invite-based account registration](https://blog.prosody.im/great-invitations/) which can help guide new users through choosing a suitable client and registering their first XMPP account. Finally, they published a round-up of advice for everyone on how to [help address spam on the XMPP network](https://blog.prosody.im/simple-anti-spam-tips/).
```

Comment:
The Prosody XMPP server team **have** been -> The Prosody XMPP server team **has** been